README.md

```diff
@@ -28,8 +28,83 @@ Please refer to [Heads-wiki](https://osresearch.net) for your Heads' documentati
 
 Building heads
 ===
+
+Under QubesOS?
+====
+* Setup nix persistent layer under QubesOS (Thanks @rapenne-s !)
+  * https://dataswamp.org/~solene/2023-05-15-qubes-os-install-nix.html
+* Install docker under QubesOS (imperfect old article of mine. Better somewhere?)
+  * https://gist.github.com/tlaurion/9113983bbdead492735c8438cd14d6cd
+
+Build docker from nix develop layer locally
+====
+
+```
+# DANGER: remove /nix store and recreates a fresh one. Skip if you use Nix already:
+sudo rm -rf /nix/* || echo "cannot delete /nix" &&  sh <(curl -L https://nixos.org/nix/install) --no-daemon
+# Configure nix for local builds for nix-commands and flakes usage under nix which are considered experimental features
+mkdir -p ~/.config/nix
+echo 'experimental-features = nix-command flakes' >~/.config/nix/nix.conf
```

Comment:
@mmlb sufficient then in link o your comment https://github.com/linuxboot/heads/pull/1661#issuecomment-2102938642 ?

Reply:
@JonathonHall-Purism should be fixed under [181ce62](https://github.com/linuxboot/heads/pull/1661/commits/181ce621bb913d3fd8ed8f9c13222850555f6615)

Reply:
TBH....this feels much more complicated (and error-prone), now we have three sections and possibly more paths than that (first step still says "skip if you use Nix already" creating more options).  They are so close it feels like we could combine them.  What about:

```
Build docker from nix develop layer locally
====

#### Set up Nix and flakes

* If you don't already have Nix, install it:
    * `[ -d /nix ] || sh <(curl -L https://nixos.org/nix/install) --no-daemon`
    * `. /home/user/.nix-profile/etc/profile.d/nix.sh`
* Enable flake support in nix
    * `mkdir -p ~/.config/nix`
    * `echo 'experimental-features = nix-command flakes' >>~/.config/nix/nix.conf`

#### Build image

* Build nix developer local environment with flakes locked to specified versions
    * `nix --print-build-logs --verbose develop --ignore-environment --command true`
* Build docker image with current develop created environment (this will take a while and create "linuxboot/heads:dev-env" local docker image:
    * `nix build .#dockerImage && docker load < result`

Done!
```

That's just about half the length, captures everything, and is easier to follow I think.  I simplified some of the descriptions a bit, moved the nix.sh sourcing right after install, and used `>>` when appending nix.conf.

Reply:
BTW, what if somebody has previously installed nix without using `--no-daemon`, does it work or would they need to reinstall?

Reply:
> BTW, what if somebody has previously installed nix without using --no-daemon, does it work or would they need to reinstall?

@JonathonHall-Purism from discussions I had, the installation method doesn't interfere on usage.